draft-ietf-quic-transport.md

```diff
@@ -2357,11 +2361,11 @@ shown in the following figure and described below.
                             +--------+
                                  |
                         send/recv STREAM/RST
-                             recv MSD/SB
+                             recv MSD/SB             
                                  |
                                  v
-                 recv FIN/  +--------+    send FIN/
-                 recv RST   |        |    send RST
+              recv FIN/     +--------+     send FIN/
+         recv RST/send UNI  |        |  send RST/recv UNI
```

Comment:
also if a RST and STREAM frame for a unidirectional stream are reordered so that RST is received first, the receiver will go into half closed remote state and have no idea that they can't send data in this state.